### PR TITLE
Add CI workflows

### PR DESCRIPTION
**Pull request by SaikrishnaBairamoni**

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (dmccoystephenson) [inline .github/workflows/ci.yml:10]: There are some incoming CDOT changes that replace ubuntu with alpine as the base image for the ACM. This will need to be modified once those changes have been merged into USDOT develop.

File: .github/workflows/ci.yml
Comment (dmccoystephenson) [inline .github/workflows/ci.yml:56]: All references to apt will need to be replaced with apk-equivalent statements when the incoming CDOT alpine-image changes are merged into USDOT develop.

File: build.sh
Comment (dmccoystephenson) [inline build.sh:28]: Has this script been tested and verified to be working as expected?

**Review by dmccoystephenson — COMMENTED** [review]
There is an open CDOT PR (https://github.com/CDOT-CV/asn1_codec/pull/14) that involves switching out ubuntu as the base image for the ACM with alpine. These changes look good for the current base image, but once these changes are in USDOT develop this will need to be modified.

**Review by dmccoystephenson — APPROVED** [review]
The Alpine base image switch probably won't be part of the Q3 release, so it seems like a good idea to merge this now. We can deal with the package manager modifications at a later time.

**Review by TonyEnglish — APPROVED** [review]
The Alpine image work will wait until the next release for Q4, this is good to merge.

**Pull request merged**
